with-emotion/popup.tsx

```diff
@@ -0,0 +1,49 @@
+import { useState, useMemo } from "react"
+import createCache from "@emotion/cache";
+import styled from "@emotion/styled";
+import { CacheProvider } from "@emotion/react";
+
+const Container = styled.main`
+  display: flex;
+  flex-direction: column;
+  padding: 1rem;
+`;
+
+const Link = styled.a`
+  padding: 0.25rem;
+  color: cornflowerblue;
+`;
+
+function IndexPopup() {
+  const [data, setData] = useState("");
+  const styleCache = useMemo(() => {
+    const $mountPoint = document.getElementById("root");
+    const cache = createCache({

```

Comment:
This is not needed for popup I think, the cache will mount automatically into the body of the popup